spec.html

```diff
@@ -228,7 +228,9 @@ <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _res
           1. Set _iteratorRecord_.[[Done]] to *true*.
           1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
           1. If _remainingElementsCount_.[[Value]] is 0, then
-            1. Perform ? ThrowAggregateError(_errors_).
+            1. Let _error_ be a newly-created `AggregateError` object.
```

Comment:
```suggestion
        1. Let _error_ be a newly created `AggregateError` object.
```

(Every other use of "newly created" in the spec is spelled this way; cc @mathiasbynens.)

Reply:
Sure, but that’s wrong.

Reply:
I think proposals should follow the style conventions in the upstream spec. If you want it to change here, it should change upstream first.

Reply:
https://github.com/tc39/ecma262/pull/1785